website/docs/r/search_service.html.markdown

```diff
@@ -52,6 +52,8 @@ The following arguments are supported:
 
 -> **Note:** `partition_count` and `replica_count` can only be configured when using a `standard` sku.
 
+* `ip_rules` - (Optional) A list of IP addresses that are allowed access to the search service endpoint. 
```

Comment:
Should we have an example of whats expected? eg `10.0.0.1` ?

Reply:
We have some restrictions with loop/local addresses so I'd have to put some random IP there which doesn't feel great. I'll just update that it's a list of ipv4 addresses and we can update it further if there is confusion.